### PR TITLE
TEST-1234: added second class

### PR DESCRIPTION
**Pull request by bajajm208**

### CODE REVIEW

File: src/com/test/SecondFile.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.test;
+
+public class SecondFile {
+
+	public static void main(String[] args) {
+		System.out.print("Second File");
```
Comment (ankitbajaj66) [inline src/com/test/SecondFile.java]: Please change message to next file.

File: .metadata/.plugins/org.eclipse.egit.core/.org.eclipse.egit.core.cmp/.project [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>.org.eclipse.egit.core.cmp</name>
+	<comment></comment>
+	<projects>
+	</projects>
```
Comment (ankitbajaj66) [inline .metadata/.plugins/org.eclipse.egit.core/.org.eclipse.egit.core.cmp/.project:6]: please check this

**Review by ankitbajaj66 — CHANGES_REQUESTED** [review]

File: src/com/test/SecondFile.java (hunk 1 above)
Comment (bajajm208) [reply src/com/test/SecondFile.java]: Done

File: .metadata/.plugins/org.eclipse.egit.core/.org.eclipse.egit.core.cmp/.project (hunk 2 above)
Comment (bajajm208) [reply .metadata/.plugins/org.eclipse.egit.core/.org.eclipse.egit.core.cmp/.project:6]: As discussed Not needed

**Review by ankitbajaj66 — APPROVED** [review]

**Pull request merged**